src/services/postService.js

```diff
@@ -41,7 +56,7 @@ module.exports = {
 
   getPostsOfAuthors: function(authors) {
     const authorObjs = authors.map(name => {
-      return { id: name };
+      return { name };
```

Comment:
This could be
```
const authorObjs = authors.map(name => { name });
```

I think this change was made in a different PR as well. I remember making this comment and making one about naming .

Reply:
Cool, made these changes in a differne tpr